Adding usage tip for ignore cpu affinity (#1948)
ROCm/rccl: docs/how-to/rccl-usage-tips.rst

This feature requires GPUs that support peer-to-peer access along with
proper large BAR addressing support.

Ignoring CPU affinity with multi-node
=====================================

Depending on the job launcher and the requirements of your workload, performance as communication scales
can be improved by setting ``NCCL_IGNORE_CPU_AFFINITY`` which allows the RCCL communication library to 

> ignore the job's supplied CPU affinity and use the GPU affinity only.

Reviewer: ```suggestion
Depending on the job launcher and the requirements of your workload, performance as the communication workload scales
can be improved by setting ``NCCL_IGNORE_CPU_AFFINITY``.  This allows the RCCL communication library to 
ignore the job's supplied CPU affinity and use the GPU affinity only.
```

.. code-block:: shell

   NCCL_IGNORE_CPU_AFFINITY=1

> For the general case, this environment variable is not set so as to not interfere with the user or launcher

Reviewer: ```suggestion
For general usage, this environment variable is not set so it doesn't interfere with the user or launcher
```
